src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassServiceUtil.java

```diff
@@ -16,6 +16,8 @@
 
 public class CafcassServiceUtil {
 
+    public static final String CAFCASS = "CAFCASS";
```

Comment:
multiple usage found. Therefore move this to PrlAppsConstants and use it in both the files.